src/main/java/me/drepic/proton/RabbitMQManager.java

```diff
@@ -55,7 +55,8 @@ protected void connect() throws IOException, TimeoutException {
         channel.exchangeDeclare("proton.direct", "headers");
 
         queueName = channel.queueDeclare().getQueue();
-        channel.basicConsume(queueName, true, this::deliverCallback, consumerTag -> {});
+        channel.basicConsume(queueName, true, this::deliverCallback, consumerTag -> {
```

Comment:
style: please maintain the existing style of this code of having an empty lambda take up only a single line